### DISCUSSION TITLE
OpenTracing JDBC and reactive sql clients

### DISCUSSION THREAD

**Discussion by GeauxEric** [quarkusio/quarkus#37508] [Q&A]
I am using reactive sql clients (https://quarkus.io/guides/reactive-sql-clients#common-datasource) + opentracing jdbc driver (https://github.com/opentracing-contrib/java-jdbc) to query against postgres.

I found that tracing is not working with reactive sql clients, while my colleagues using regular sql datasource have the tracing working.

Is that expected that reactive sql clients do not have tracing supported?

My quarkus platform version: 1.13.6.Final

[a comment by an automated account is not shown]

**Comment by vietj** [comment]
vertx client do implement tracing as far as I know, see https://github.com/eclipse-vertx/vertx-tracing/blob/master/vertx-opentelemetry/src/test/java/io/vertx/tracing/opentelemetry/SqlClientTest.java

**Reply by GeauxEric** [reply]
may I ask since which quarkus version is this supported?

**Comment by brunobat** [comment]
[GeauxEric](https://github.com/GeauxEric) OpenTracing is no longer supported in main Quarkus and no new features will be added. Please consider migrate to OpenTelemetry by checking this tutorial: https://quarkus.io/guides/telemetry-opentracing-to-otel-tutorial
